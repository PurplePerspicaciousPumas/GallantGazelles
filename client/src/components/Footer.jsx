import React, { Component } from 'react';
import { Divider, Grid, Header, Icon, Input, Menu } from 'semantic-ui-react';
import { Link } from 'react-router-dom';

export default class Footer extends Component {
  constructor(props) {
    super(props)
    this.state = { activeItem: 'home' };

    this.handleItemClick = (e, { name }) => this.setState({ activeItem: name });
  }


  render () {
    const { activeItem } = this.state;

    return (
      <div>
        <Divider hidden />
        <Divider />

        <Grid padded container columns='2'>

          <Grid.Column floated='left' verticalAlign='middle'>
            <Link to="/">PitchMe.io</Link>
          </Grid.Column>

          <Grid.Column floated='right' verticalAlign='middle'>
            <Menu text>
              <Menu.Menu position='right'>
                <Menu.Item name='home' as={Link} to='/' active={activeItem === 'home'} onClick={this.handleItemClick} />
                <Menu.Item name='how it works' as={Link} to='/howitworks' active={activeItem === 'how it works'} onClick={this.handleItemClick} />
                <Menu.Item name='startups' as={Link} to='/startups' active={activeItem === 'startups'} onClick={this.handleItemClick} />
              </Menu.Menu>
            </Menu>
          </Grid.Column>

        </Grid>
      </div>
    )
  }
}