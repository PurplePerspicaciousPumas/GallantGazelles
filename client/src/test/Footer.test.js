import React from 'react';
import Footer from '../components/Footer.jsx';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <Footer />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});