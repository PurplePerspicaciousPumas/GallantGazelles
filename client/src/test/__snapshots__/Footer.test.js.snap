// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer>
  <div>
    Copyright Gallant Gazelles 2017
  </div>
  <div>
    <span>
      Home 
    </span>
    <span>
      About Us 
    </span>
    <span>
      FAQ 
    </span>
    <span>
      Privacy Policy 
    </span>
    <span>
      Support
    </span>
  </div>
</footer>
`;
